File: app/src/main/java/in/aerem/comconbeacons/ComConBeaconsApplication.java
package in.aerem.comconbeacons;

import android.app.Application;
import android.content.SharedPreferences;
import android.preference.PreferenceManager;

public class ComConBeaconsApplication extends Application {
    public SharedPreferences getGlobalSharedPreferences() {
        return PreferenceManager.getDefaultSharedPreferences(this);
    }
}

File: app/src/main/java/in/aerem/comconbeacons/ComConBeaconsApplication.kt
package `in`.aerem.comconbeacons

import android.app.Application
import android.content.SharedPreferences
import android.preference.PreferenceManager
import com.bugfender.sdk.Bugfender

class ComConBeaconsApplication : Application() {
    fun getGlobalSharedPreferences(): SharedPreferences {
        return PreferenceManager.getDefaultSharedPreferences(this)
    }

    override fun onCreate() {
        super.onCreate()
        Bugfender.init(this, "LqR51nrg2rygHhlHefCwf0HIgG4F04DB", BuildConfig.DEBUG)
        Bugfender.enableCrashReporting()
        Bugfender.enableUIEventLogging(this)
        Bugfender.enableLogcatLogging()
    }
}
